client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/metrics/DefaultNoopMetricsRecorder.java

```diff
@@ -12,4 +12,9 @@ public class DefaultNoopMetricsRecorder implements MetricsRecorder {
   public void increase(String metricName, String action, String type) {
     // ignore
   }
+
+  @Override
+  public void executeWithTimer(String metricName, Runnable methodToExecute) {
+    // ignore
```

Comment:
Should we add `methodToExecute.run()` here? Otherwise it seems like the method won't be executed at all.

Reply:
Very well spotted, thank you 👍 I fixed it.